scripts/test/run-tests.sh

```diff
@@ -48,58 +48,61 @@ mkdir -p ${WORKSPACE}
 
 get_pytest_files
 
+if ! [ "$1" == "--all" -o "$1" == "--only" -a -n "$2" -o "$1" == "--prebuilt-image" -a -n "$3" ]; then
```

Comment:
The argument parsing isn't very good in this script (depends on position), but it was preexisting.